### PR TITLE
add cassandra testcases

### PR DESCRIPTION
**Pull request by stone-wlg**
Please fill up those following information before send Pull request 

* Pull request NO: https://github.com/apache/skywalking/pull/3410  #3410
* Agent repository URL: https://github.com/stone-wlg/skywalking.git  branch:features/cassandra-java-driver-3.x-plugin
* Testcases repository URL: https://github.com/stone-wlg/agent-auto-integration-testcases.git  branch:master

### CODE REVIEW

**Comment by zhaoyuguang** [comment]
Pull request NO: https://github.com/apache/skywalking/pull/3330

File: cassandra-3.x-scenario/config/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  mock-collector:
+    image: skyapm/mock-collector:{COLLECTOR_IMAGE_VERSION}
+    environment:
+      - GRPC_BIND_HOST=mock-collector
+    expose:
+      - "12800"
+    ports:
+      - "{COLLECTOR_OUTPUT_PORT}:12800"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://127.0.0.1:12800/receiveData"]
```
Comment (wu-sheng) [inline cassandra-3.x-scenario/config/docker-compose.yml]: Have you used this to check server status?

Comment (stone-wlg) [reply cassandra-3.x-scenario/config/docker-compose.yml]: no, how to use it to check server status ? is there a sample to follow ?

**Comment by stone-wlg** [comment]
[INFO] ---------------------------Here is the test report context -------------------------------------

# Test Report - 2019-08-30 10:38

- tests  : **4 passed**. **0 failed**
- branch name : **[features/cassandra-3.x-plugin](https://github.com/apache/incubator-skywalking/tree/features/cassandra-3.x-plugin)**
- commit id : **[b1d0b0521b5c88ae2b2ccbf65e8edd0fd50148f0](https://github.com/apache/incubator-skywalking/commit/b1d0b0521b5c88ae2b2ccbf65e8edd0fd50148f0)**
- cases branch: **[master](https://github.com/SkywalkingTest/skywalking-autotest-scenarios/tree/master)**
- cases commit id: **[240385c35b88c896c96ba801d21064e78b54d0f9](https://github.com/SkywalkingTest/skywalking-autotest-scenarios/commit/240385c35b88c896c96ba801d21064e78b54d0f9)**

## Cases List

| Case description | Status | Cases|
|:-----|:-----:|:-----:|
|Cassandra| **4 passed. 0 failed**| [click me](#cassandra) |

## Cassandra

### 
|  Version     | Passed | Failed|
|:------------- |:-------:|:-----:|
| 3.6.0  | :heavy_check_mark:||
| 3.7.2  | :heavy_check_mark:||
| 3.7.0  | :heavy_check_mark:||
| 3.7.1  | :heavy_check_mark:||


[INFO] -----------------------------------end the report-----------------------------------------------

**Comment by stone-wlg** [comment]
[INFO] ---------------------------Here is the test report context -------------------------------------

# Test Report - 2019-09-04 16:57

- tests  : **1 passed**. **0 failed**
- branch name : **[features/cassandra-java-driver-3.x-plugin](https://github.com/apache/incubator-skywalking/tree/features/cassandra-java-driver-3.x-plugin)**
- commit id : **[0977457e42aeec72ff5efb722f6ddc4673d84864](https://github.com/apache/incubator-skywalking/commit/0977457e42aeec72ff5efb722f6ddc4673d84864)**
- cases branch: **[master](https://github.com/SkywalkingTest/skywalking-autotest-scenarios/tree/master)**
- cases commit id: **[a3d32e1e6385660f0cefd04bfc580b45f5aebeaf](https://github.com/SkywalkingTest/skywalking-autotest-scenarios/commit/a3d32e1e6385660f0cefd04bfc580b45f5aebeaf)**

## Cases List

| Case description | Status | Cases|
|:-----|:-----:|:-----:|
|Cassandra| **1 passed. 0 failed**| [click me](#cassandra) |

## Cassandra

### 
|  Version     | Passed | Failed|
|:------------- |:-------:|:-----:|
| 3.7.2  | :heavy_check_mark:||


[INFO] -----------------------------------end the report-----------------------------------------------

**Comment by stone-wlg** [comment]
[INFO] ---------------------------Here is the test report context -------------------------------------

# Test Report - 2019-09-09 13:30

- tests  : **4 passed**. **0 failed**
- branch name : **[features/cassandra-java-driver-3.x-plugin](https://github.com/apache/incubator-skywalking/tree/features/cassandra-java-driver-3.x-plugin)**
- commit id : **[7e6124170f6e5d58f12d03c9a752f77c92480d5d](https://github.com/apache/incubator-skywalking/commit/7e6124170f6e5d58f12d03c9a752f77c92480d5d)**
- cases branch: **[master](https://github.com/SkywalkingTest/skywalking-autotest-scenarios/tree/master)**
- cases commit id: **[565d60c983a2c527f17997bd5cf8aa6eda30c0be](https://github.com/SkywalkingTest/skywalking-autotest-scenarios/commit/565d60c983a2c527f17997bd5cf8aa6eda30c0be)**

## Cases List

| Case description | Status | Cases|
|:-----|:-----:|:-----:|
|Cassandra| **4 passed. 0 failed**| [click me](#cassandra) |

## Cassandra

### 
|  Version     | Passed | Failed|
|:------------- |:-------:|:-----:|
| 3.7.1  | :heavy_check_mark:||
| 3.7.2  | :heavy_check_mark:||
| 3.7.0  | :heavy_check_mark:||
| 3.6.0  | :heavy_check_mark:||


[INFO] -----------------------------------end the report-----------------------------------------------

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**